kili/helpers.py

```diff
@@ -90,12 +90,16 @@ def format_json(result):
     if isinstance(result, dict):
         for key, value in result.items():
             if key in ['jsonInterface', 'jsonMetadata', 'jsonResponse']:
-                if value == '' or value is None:
+                if key == 'jsonMetadata' and is_url(result[key]):
+                    print('escape')
+                    result[key] = result[key]
```

Comment:
First part of the `if` could be skipped?

Reply:
What do you mean ? We can do only the check is_url(value) ?